packages/sdk/src/restclient.spec.ts

```diff
@@ -188,13 +198,40 @@ describe("RestClient", () => {
   });
 
   describe("authAccounts", () => {
-    it("works", async () => {
+    it("works for unused account without pubkey", async () => {
+      pendingWithoutCosmos();
+      const client = new RestClient(httpUrl);
+      const { result } = await client.authAccounts(unusedAccount.address);
+      expect(result).toEqual({
+        type: "cosmos-sdk/Account",
+        value: {
+          address: unusedAccount.address,
+          public_key: "", // not known to the chain
```

Comment:
Question

I think i did a parse step here, so pubkey is always the decoded format, not the bech32 format. Which is much easier to use/check in all other code.

And now I think that code was in a pr I closed without merge. Anyway, I would like this to be an object or undefined. But I can do that separate

Reply:
Such a parsing step exists in BCP CosmWasmConnection: `const pubkey = !account.public_key ? undefined : decodeCosmosPubkey(account.public_key);`

At this level I tend to pass through the REST response 1:1. However, I think there should be a higher CosmWasmClient around RestClient without BCP types that exposes the perfect TypeScript types.

Reply:
Okay, I agree with this.

Rest server 1:1, bcp adapter for compatibility with tooling. And a to.be created higher level sdk for using one cosmwasm chain.

Reply:
A part of my inspiration is that the rest server types keep changing (more will be bechified, etc.) And want to give a relatively stable api to apps. That can be done in another class.